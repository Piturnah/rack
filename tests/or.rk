true false or if
  1 print
end

false true or if
  1 print
end

true true or if
  1 print
end

false false or if
  0 print
end