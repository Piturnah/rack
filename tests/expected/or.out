----STDOUT----
1
1
1

----STDERR----
